### PR TITLE
Update materials

### PR DESCRIPTION
**Pull request by dmenneck**
Updates the contents of the workshop for the upcoming FOSSGIS.

### CODE REVIEW

File: src/basics/installextensions.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Führen Sie die folgenden Schritte aus, um diese Erweiterung zu installieren:
 </xmp></pre>
 1. Da Sie root-Rechte benötigen, um in das lib-Verzeichnis schreiben zu können,
    muss der folgende Befehl zum Entpacken des Archivs mit sudo ausgeführt werden:
-<pre><xmp style="margin:0; font-size: .85em;">sudo unzip /home/user/Downloads/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
+<pre><xmp style="margin:0; font-size: .85em;">sudo unzip /home/user/Downloads/geoserver-2.13.2-wps-plugin.zip
```
Comment (dnlkoch) [inline src/basics/installextensions.md]: Is the variable not working here (and above)? It should be `2.15.1` either way.

File: q [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+[33m33a28cd[m Output to dev/null
+[33m7fac7e6[m CI changes
+[33mf98da16[m Make script executable
+[33m045124a[m ci test
+[33m5c47a15[m Update security token
+[33m3b8fd68[m Add simple SLD resource
+[33mc276890[m Remove git submodule init as no submodules present
+[33m9705be6[m Add travis config
+[33mcf64a13[m Initial upload
+[33md950069[m Initial commit
```
Comment (dnlkoch) [inline q]: I assume, this file isn't intended to be merged?

**Review by dnlkoch — CHANGES_REQUESTED** [review]

File: src/basics/installextensions.md (hunk 1 above)
Comment (dmenneck) [reply src/basics/installextensions.md]: The [link](https://raw.githubusercontent.com/terrestris/geoserver-in-action-ws/master/src/materials/geoserver-2.13.2-wps-plugin.zip) to download this zip-file (geoserver-2.13.2-wps-plugin.zip) only works with `2.13.2` and not with `2.15.1`. The variable *book.geoServerVersion* would change the download link to https://raw.githubusercontent.com/terrestris/geoserver-in-action-ws/master/src/materials/geoserver-2.15.1-wps-plugin.zip which does not work.

File: q (hunk 2 above)
Comment (dmenneck) [reply q]: Yes, this file should not be merged.

**Pull request merged**